prediction-applier-extension/src/main/resources/alfresco/module/alfresco-hxinsight-connector-prediction-applier-extension/context/service-context.xml

```diff
@@ -19,4 +19,52 @@
        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:schemaLocation="http://www.springframework.org/schema/beans
           http://www.springframework.org/schema/beans/spring-beans-3.0.xsd">
+
+    <bean id="predictionService" class="org.alfresco.hxi_connector.prediction_applier.service.PredictionServiceImpl">
+        <property name="nodeService" ref="NodeService"/>
```

Comment:
Could these two be `constructor-arg`s (see one of my previous comments)?

Reply:
Done